packages/cli/test/commands/migrate.test.ts

```diff
@@ -216,15 +226,15 @@ describe('migrate - handle custom basePath', async () => {
 
   test('Run cli againt specific basePath via -basePath option', async () => {
     const customBasePath = resolve(basePath, 'base');
-    const result = await runBin('migrate', ['--basePath', customBasePath], {
+    const result = await runBin('migrate', ['--basePath', customBasePath, '-v'], {
       cwd: basePath, // still run cli in basePath
```

Comment:
`cwd: basePath` is causing failures ... if you rebase this will/should be removed